docs/content/dagster-plus/deployment/azure/blob-compute-logs.mdx

```diff
@@ -0,0 +1,119 @@
+# Storing compute logs in Azure Blob Storage/Azure Data Lake Storage
+
+In this guide, we'll walk through how to store compute logs in Azure Blob Storage or Azure Data Lake Storage. This guide assumes you have already set up an Azure Kubernetes Service (AKS) agent and deployed user code in Azure Container Registry (ACR).
+
+This guide focuses on using Azure Blob Storage, but the same steps should be applicable for Azure Data Lake Storage.
+
+If you have not yet set up an AKS agent, you can follow the [Deploy an Azure Kubernetes Service (AKS) agent guide](/dagster-plus/deployment/azure/aks-agent).
+If you have not yet deployed user code in ACR, you can follow the [Deploy user code in Azure Container Registry (ACR) guide](/dagster-plus/deployment/azure/acr-user-code).
+
+## Prerequisites
+
+To complete the steps in this guide, you'll need:
+- The Azure CLI installed on your machine. You can download it [here](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli).
+- An Azure account with the ability to create resources in Azure Blob Storage or Azure Data Lake Storage.
+- An Azure container in Azure Blob Storage or Azure Data Lake Storage where you want to store logs.
+
+## Step 1: Give AKS agent access to blob storage account
+
+We need to ensure that the AKS agent has the necessary permissions to write logs to Azure Blob Storage or Azure Data Lake Storage. How
+ We'll do this with some azure CLI commands.
+
+First, we'll enable the cluster to use workload identity. This will allow the AKS agent to use a managed identity to access Azure resources.
+
+```bash
+az aks update --resource-group <resource-group> --name <cluster-name> --enable-workload-identity
+```
+
+Then, we'll create a new managed identity for the AKS agent, and a new service account in our AKS cluster.
+
+```bash
+az identity create --resource-group <resource-group> --name agent-id
+kubectl create serviceaccount dagster-agent-sa --namespace dagster-agent 
+```
+
+Now we need to federate the managed identity with the service account.
+
+```bash
+az identity federated-credential create \                                                                             
+  --name dagster-agent-federated-id \
+  --identity-name agent-id \
+  --resource-group <resource-group> \
+  --issuer $(az aks show -g <resource-group> -n <aks-cluster-name> --query "oidcIssuerProfile.issuerUrl" -otsv) \
+  --subject system:serviceaccount:dagster-agent:dagster-agent-sa
```

Comment:
i think longer identifiers would make it more explicit and grokkable for the user.

agent-id -> agent-identity
dagster-agent-sa -> dagster-agent-service-account